### DISCUSSION TITLE
How do we maintain the Authors list over multiple revisions?

### DISCUSSION THREAD

**Discussion by ahouseholder** [CERTCC/SSVC#280] [General]
I don't know how this is handled in publishing, scientific or otherwise. But with SSVC, we have a document that has gone through multiple published revisions over 4 years now, and many of the previous authors have moved on to other employers and are no longer involved in the development of the document and associated resources. Obviously we need to continue to acknowledge their contribution to past versions, but I'm concerned that continuing to include them as authors on future revisions may be misrepresenting both their implied approval and contribution to versions they had no hand in creating.  It's not clear to me what a sustainable process for this over time looks like.

This probably ties in with versioning discussions as well, because I could imagine a process that looks like:
- add authors to "minor" versions (no removals)
- at each "major" version
  - sweep the authors list into the Acknowledgements section
  - adjust the Authors to reflect those who specifically contributed to that version

I'm not saying that's the best process, it's just one I thought up as I was writing this post. Curious what others think, and what known precedents are out there.

**Comment by j---** [comment]
We could also remove the authors from the title page and just list the people that contributed to each time period / release version. 

Something like:

> Contributors to SSVC v1.0 are:
...
Contributors to SSVC v1.1 are:
...
Contributors to SSVC 2.0 are:
...
Contributors to SSVC 2.1 are:
...

If we do this, we end up only presenting the whole historical context, which is too messy to fit on an author byline and makes citing the document harder. But it also preserves all the contributions in a stable way. But I can't think of a good way to recommend how to cite SSVC then as far as authors. Though I suppose how it's cited could be the cited version's authors, even if those are not front and center at the start of the PDF.

**Comment by jeroenh** [comment]
Usually for technical documents or standards that are created by a working group there is a (couple of) editor(s), and then contributors are listed separately. And for different versions you can have different editors, but those tend to be somewhat more stable.

**Comment by ahouseholder** [comment]
I've got an inquiry out to our in-house folks who have more experience in long-lived multi-version information products as well, I'll update if any additional advice surfaces on that front.

**Comment by ahouseholder** [comment]
Addressing this in issue #297 and pr #298
